rootbeerlib/build.gradle.kts

```diff
@@ -77,73 +53,50 @@ dependencies {
     testImplementation(libs.mockito)
 }
 
-// helper method to ensure we have a non null string for a property
-fun getPropertyOrEmpty(propertyName: String): String = project.findProperty(propertyName)?.toString().orEmpty()
-
-project.version = getPropertyOrEmpty("VERSION_NAME")
```

Comment:
I removed these because they were duplicated definitions - they are already defined in the root `build.gradle.kts` file.